processor/build.gradle

```diff
@@ -0,0 +1,10 @@
+apply plugin: 'java'
+
+dependencies {
+    compile project(":annotations")
+    compile 'com.squareup:javapoet:1.0.0'
+
+    testCompile 'junit:junit:4.12'
+    testCompile 'com.google.testing.compile:compile-testing:0.6'
+    testCompile 'com.google.android:android:4.1.1.4'
+}
```

Comment:
Please add a trailing newline to prevent future changes from having this line show up in a diff as changed.